### PR TITLE
Bump version kubernetes split yaml 0.3.0

### PR DESCRIPTION
**Pull request by edlimerkaj**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/kubernetes-split-yaml/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  };
+
+  subPackages = ["."];
+
+  vendorSha256 = "0gxvzz5mx84jpanpp9cv7hv1h2j6w4zy42h4wb1zxlidafx3x95x";
+
+  meta = with lib; {
+    description = "Split the 'giant yaml file' into one file pr kubernetes resource";
+    license = licenses.mit;
+    maintainers = with maintainers; [ "edlimerkaj" ];
```
Comment (mohe2015) [inline pkgs/development/tools/kubernetes-split-yaml/default.nix:21]: You need to add yourself to maintainers-list e.g like I did in https://github.com/NixOS/nixpkgs/pull/112322/commits/a3141361cbfc53ca2c45f37a59cf1de83a19d9ef

**Comment by mohe2015** [comment]
Please remove the first two commits as they seem to just revert themself. If you need help with that feel free to ask how to do this.

File: pkgs/development/tools/kubernetes-split-yaml/default.nix (hunk 1 above)
Comment (mohe2015) [reply pkgs/development/tools/kubernetes-split-yaml/default.nix:21]: Please do this in a separate commit.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -25288,6 +25288,8 @@ in
 
   kubernetes-helm = callPackage ../applications/networking/cluster/helm { };
 
+  kubernetes-split-yaml = callPackage ../development/tools/kubernetes-split-yaml {  };
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:25291]:
```suggestion
  kubernetes-split-yaml = callPackage ../development/tools/kubernetes-split-yaml { };
```

File: pkgs/development/tools/kubernetes-split-yaml/default.nix [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    rev = "v${version}";
+    sha256 = "0b6ivmznbslcqp780v91qc7mrgydzkhf8jbjlwdn1k8rffkwynnd";
+  };
+
+  subPackages = ["."];
+
+  vendorSha256 = "0gxvzz5mx84jpanpp9cv7hv1h2j6w4zy42h4wb1zxlidafx3x95x";
+
+  meta = with lib; {
+    description = "Split the 'giant yaml file' into one file pr kubernetes resource";
```
Comment (SuperSandro2000) [inline pkgs/development/tools/kubernetes-split-yaml/default.nix:19]: Missing homepage.

**Review by SuperSandro2000 — COMMENTED** [review]
Also squash the commits according to the contributing guide.

File: maintainers/maintainer-list.nix [hunk 4]
```diff
@@ -3007,6 +3007,13 @@
...
     name = "Eduardo Sánchez Muñoz";
   };
+  edlimerkaj = {
+      name = "Edli Merkaj";
+      email = "edli.merkaj@identinet.io";
+      github = "edlimerkaj";
+      githubId = 71988351;
+  };
+
   eduarrrd = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:3017]:
```suggestion
  };
  eduarrrd = {
```

**Pull request closed without merging**